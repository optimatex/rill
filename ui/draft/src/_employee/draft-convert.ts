export { convertToHTML, convertFromHTML } from 'draft-convert';
