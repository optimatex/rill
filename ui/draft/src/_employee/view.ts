export {
  React,
  useState,
  useCallback,
  css,
  styled,
  createGlobalStyle,
} from '@rill/view';
