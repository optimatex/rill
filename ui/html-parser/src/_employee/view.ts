export { React, RC } from '@rill/view';
