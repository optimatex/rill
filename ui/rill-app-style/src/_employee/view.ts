export { createGlobalStyle, css } from '@rill/view';
