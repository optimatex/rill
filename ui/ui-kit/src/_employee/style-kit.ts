export {
  // appLayout
  appLayoutCss,
  screenCss,
  sidebarColumnCss,
  flatColumnCss,
  mainColumnCss,
  // layout
  listCss,
  // Brand
  logCss,
  dualLogoCss,
  // Controller
  trivialAppHeaderCss,
  // _menu
  menuWrapCss,
  menuItemCss,
  // Image
  imageCss,
  imagesListCss,
  imagesGridCss,
  imagesLayoutItemCss,
  imageCropperCss,
  // Tags
  tag,
  tagsList,
  // Ui-primal
  // _grid
  gridRowContainerCss,
  gridrillRowContainerCss,
  gridColumnCss,
  // _card
  cardContainerCss,
  postContainerCss,
  postTitleCss,
  panelCss,
  // _buttons
  buttonCss,
  buttonActiveCss,
  buticonCss,
  // _form
  formContainerCss,
  formFieldFlatCss,
  formFieldColumnCss,
  formFieldErrorCss,
  formLabelTextCss,
  inputCss,
  toggleContainerCss,
  toggleContainerActiveCss,
  toggleRoundCss,
  toggleRoundActiveCss,
  // _article
  paragraphCss,
  nameCss,
  richContentWrapCss,
  // _table
  tableContainerCss,
  tableHeadCss,
  tableRowCss,
  tableColumnCss,
  // Unknown
  // _object
  stringValueCss,
  numberValueCss,
  booleanValueCss,
  emptyValueCss,
  arrayWrapCss,
  arrayItemCss,
  objectKeyCss,
  objectWrapCss,
  objectPairCss,
  // Unsorted
  lightIndicatorCss,
  popoverCss,
  pipeCss,
  hintableLogCss,
  fixedPanelCss,
} from '@rill/style-kit';
