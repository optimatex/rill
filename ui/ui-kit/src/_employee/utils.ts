export {
  toPairs,
  find,
  has,
  path,
  prop,
  curry,
  assoc,
  lensProp,
  view,
  set,
} from '@rill/utils';
