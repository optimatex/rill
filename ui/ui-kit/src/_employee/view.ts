export {
  React,
  useState,
  useCallback,
  // styled-components
  styled,
  // Lens
  // getTestId,
  // Types
  FC,
  RC,
  ReactNode,
  LegacyRef,
  // PropTypes
  IContainerProps,
  // Event types
  FormEvent,
  MouseEvent,
} from '@rill/view';
