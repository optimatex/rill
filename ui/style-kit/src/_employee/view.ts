import { css as originalCss } from '@rill/view';

export const css = originalCss;
