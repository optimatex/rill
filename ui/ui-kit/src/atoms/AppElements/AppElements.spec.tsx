import { React } from '_employee/view';
import { render, text_1, text_2, text_3 } from 'test';
import {
  AppLayout,
  TrivialAppLayout,
  Screen,
  SidebarColumn,
  ContentColumn,
} from '.';

describe('AppElements', () => {
  it('Screen', () => {
    const { getByText } = render(<Screen>{text_1}</Screen>);
    const el = getByText(text_1);

    expect(el).toMatchSnapshot();
  });

  it('SidebarColumn', () => {
    const { getByText } = render(<SidebarColumn>{text_2}</SidebarColumn>);
    const el = getByText(text_2);

    expect(el).toMatchSnapshot();
  });

  it('ContentColumn', () => {
    const { getByText } = render(<ContentColumn>{text_3}</ContentColumn>);
    const el = getByText(text_3);

    expect(el).toMatchSnapshot();
  });

  it('AppLayout', () => {
    const { getByText } = render(<AppLayout>{text_1}</AppLayout>);
    const el = getByText(text_1);

    expect(el).toMatchSnapshot();
  });

  it('TrivialAppLayout', () => {
    const { getByText } = render(
      <TrivialAppLayout topOffset={50}>{text_2}</TrivialAppLayout>,
    );
    const el = getByText(text_2);

    expect(el).toMatchSnapshot();
  });
});
