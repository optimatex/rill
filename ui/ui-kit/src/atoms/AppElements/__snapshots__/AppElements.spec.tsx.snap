// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppElements AppLayout 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

.c0 > * {
  margin-right: 16px;
}

.c0 > *:last-child {
  margin-right: 0;
}

<div
  class="c0"
  data-testid="layout"
>
  1 Some content 1
</div>
`;

exports[`AppElements ContentColumn 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  max-width: 560px;
}

.c0 > * {
  margin-bottom: px;
}

.c0 > *:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
  data-testid="column"
>
  3 Some content 3
</div>
`;

exports[`AppElements Screen 1`] = `
.c0 {
  background: red;
  color: white;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

.c0 > * {
  margin-bottom: px;
}

.c0 > *:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
>
  1 Some content 1
</div>
`;

exports[`AppElements SidebarColumn 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  max-width: 560px;
}

.c0 > * {
  margin-bottom: px;
}

.c0 > *:last-child {
  margin-bottom: 0;
}

<div
  class="c0"
  data-testid="column"
>
  2 Some content 2
</div>
`;

exports[`AppElements TrivialAppLayout 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

.c0 > * {
  margin-right: 16px;
}

.c0 > *:last-child {
  margin-right: 0;
}

.c0 > * {
  padding-top: 50px;
}

<div
  class="c0"
  data-testid="layout"
>
  2 Some content 2
</div>
`;
